openprescribing/matrixstore/build/dates.py

```diff
@@ -0,0 +1,39 @@
+DEFAULT_NUM_MONTHS = 60
+
+
+def generate_dates(end_str, months=None):
+    """
+    Given an end date as a string in YYYY-MM form, return a list of N
+    consecutive months as strings in YYYY-MM-01 form, with that month as the
+    final member
+    """
+    if months is None:
```

Comment:
I wanted to keep the management command as a kind of dumb wrapper around the code itself, so that it could all be invoked without involving any of the Django machinery. So for this reason I didn't want to rely on the default being set in the CLI arguments.